src/app/app.component.ts

```diff
@@ -2,12 +2,12 @@ import { Component, OnInit } from '@angular/core';
 import { AboutComponent } from './about/about.component';
 import { LoginComponent } from './login/login.component';
 import { RegistrationComponent } from './registration/registration.component';
-import { CurrentUserService } from './services/current-user.service';
-import { AuthenticationService } from './services/authentication.service';
+import { CurrentUserService } from '@services/current-user.service';
+import { AuthenticationService } from '@services/authentication.service';
 import { MatDialog } from '@angular/material/dialog';
-import { StatesService } from './services/states.service';
-import { NetworkNamesService } from './services/network-names.service';
-import { SensorTypesService } from './services/sensor-types.service';
+import { StateService } from '@app/services/state.service';
```

Comment:
why is this one "@app/services/state.service" unlike the other ones that are "@services/blah.service"?

Reply:
because of auto-import not being aware of the TS config shortcuts we have. Will fix.

Reply:
fixed